cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -438,6 +436,10 @@ func main() {
 			if len(parts) == 2 && parts[0] == "" {
 				pprofHostPort = fmt.Sprintf("localhost:%s", parts[1])
 			}
+			if !strings.Contains(pprofHostPort, "localhost") {
+				log.Printf("W! Not starting pprof, it is restricted to localhost:nnnn")
+				return
+			}
```

Comment:
This is the main change.

Everything else is just linting stuff:
1. Use lowercase for error strings.
2. remove unused `err` variable.
3. remove unused `fUsage` variable.